test/powershell/engine/Api/MultithreadedEngine.Tests.ps1

```diff
@@ -10,20 +10,23 @@ Describe 'Multithreaded engine APIs' -Tags 'CI' {
         ThreadId   = [System.Threading.Thread]::CurrentThread.ManagedThreadId
         RunspaceId = [runspace]::DefaultRunspace.Id
     }
-    Start-Sleep -Seconds 1
+    Start-Sleep -Milliseconds 500
```

Comment:
It is very unreliable.
Could we replace the sleep (here and below) with our helper function `Wait-UntilTrue` ?

Reply:
That's not true. The `Wait-UntilTrue` helper is about waiting until a certain condition occurs, or a timeout. These timers are about allowing commands to run asynchronously, and are not about waiting for a certain condition to occur. Two different use cases.

Reply:
Also just to add, I only lowered the sleep delay because I wanted the Pester tests to complete a little faster, and I didn't need a full 1 second delay in the command loop to do that.

Reply:
I made one mistake in my previous reply. The first instance (sleep) is necessary for the reasons already provided. The next one can be (and has been) replaced with `Wait-UntilTrue`.